users/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+
+from .models import Followers, User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = [
+            "username",
+            "first_name",
+            "last_name",
+            "description",
+            "date_of_birth",
+            "profile_photo",
+        ]
+
+
+class FollowersSerializer(serializers.ModelSerializer):
```

Comment:
I don't think we are going to use it. We will be showing a list of someone's followers and use `UserSerializer` for that

Reply:
Remowed